solr/prometheus-exporter/src/java/org/apache/solr/prometheus/exporter/SolrExporter.java

```diff
@@ -329,4 +355,8 @@ private static MetricsConfiguration loadMetricsConfiguration(String configPath)
       throw new RuntimeException(e);
     }
   }
+
```

Comment:
Somewhere we have code that lets you look up these settings from system variable or environment variable or solr configuration file..  That may be only applicable to main Solr code base however...!

Reply:
Yeah. The exporter has very little to do with Solr, except being shipped in the same docker (not slim) image.  I can't find any dependency to Solr core. Thus I belive that this has a completely separate configuration. 
That is why I opted to use the same name for environment variables that is used in Solr to ease the configuration of this in a production environment, eg. Kubernetes.

When I try and run `./gradlew integrationTests --tests test_ssl.bats` it fails for me (exit 1), giving a novice like me in Solr development little confidence to create more tests based on them.
`test_start_solr.bats` works fine though.
I might try and rebase on the latest from main again.

Reply:
what environment are you in?   I suppose a `test_prometheus.bats` without SSL would be good enough to start with.

I've been mulling over, should the .bats tests be moved from somewhat "unit style for scripts" type tests to a more heavyweight integration test..  i.e, fire up a SSL encyrpted Solr with SSL Zookeeper with Basic Auth enabled, and then test various things like Prometheus exporter ;-).

Reply:
There's a docker test that tests it without SSL, though it obviously could be added to bats as well

Reply:
oh, then I guess that covers it right?

Reply:
I see them now...   I wonder if would help if the docker tests used .bats?   looks like a lot of custom logic!

Reply:
I have a MacPro M3. nodejs-errorlogs are always a pain imho.

Reply:
`curl: (7) Failed to connect to localhost port 3128 after 0 ms: Couldn't connect to server`

Hmm. That is my proxy... I guess curl pics up on that.

Reply:
`unset $HTTPS_PROXY` 
Now the ssl tests run fine. Let me see what I can do with prometheus exporter.